### ISSUE TITLE
Is the version in Google Playstore newer than the one in GitHub?

### ISSUE THREAD

**Issue by VivekThazhathattil** [johnspax/AppOpener#2]
I find the app in Google Playstore has more features than the one available here. Kindly update this repo to match the newest version if that is the case.

**Comment by userkilled** [comment]
Yes, that would be nice, and it would also be nice to see it in the F-Droid or IzzyOnDroid repository
